cpp/src/barretenberg/stdlib/primitives/circuit_builders/circuit_builders.hpp

```diff
@@ -3,17 +3,22 @@
  * templates.
  */
 #pragma once
+#include "barretenberg/proof_system/circuit_builder/circuit_simulator.hpp"
 #include "barretenberg/proof_system/circuit_builder/standard_circuit_builder.hpp"
 #include "barretenberg/proof_system/circuit_builder/turbo_circuit_builder.hpp"
 #include "barretenberg/proof_system/circuit_builder/ultra_circuit_builder.hpp"
 
 template <typename T> concept HasPlookup = proof_system::IsAnyOf<T, proof_system::UltraCircuitBuilder>;
 
```

Comment:
Could unify macros, but this was easier for incrementally introducing `CircuitSimulator`.